File: mobile/src/shared/constants/limits.ts
/** 입력·초대·복약 등 도메인 한도 */
export const LIMITS = {
  inviteCodeLength: 6,
  /** 초대/복구 코드 letterSpacing */
  inviteCodeLetterSpacing: 6,
  nicknameMaxLength: 10,
  familyNameMaxLength: 20,
  /** 가족당 초대 슬롯 (DB unified invite limit과 동기) */
  maxFamilyInvites: 4,
  /** 하루 복용 시간 슬롯 상한 */
  maxTimeSlots: 6,
  /** 복약 기본 시각 (HH:MM) */
  defaultDoseTime: '08:00',
  /** TimePicker 분 단위 */
  doseMinuteInterval: 5,
  /** 약 검색 최소 글자 */
  drugSearchMinQueryLength: 2,
  /** 약 메타 텍스트 상한 (효능·복용법 등) */
  medMetaMaxLength: 2000,
  /** 용량 숫자 상한 */
  medDoseAmountMax: 9999,
} as const;

File: android-app/core/src/main/kotlin/com/jinlabs/yakok/core/constants/Limits.kt
package com.jinlabs.yakok.core.constants

/** 입력·초대·복약 한도. RN `shared/constants/limits.ts` · DB와 동기. */
object Limits {
    const val InviteCodeLength = 6
    const val InviteCodeLetterSpacing = 6
    const val NicknameMaxLength = 10
    const val FamilyNameMaxLength = 20
    const val MaxFamilyInvites = 6
    const val MaxTimeSlots = 6
    const val DefaultDoseTime = "08:00"
    const val DrugSearchMinQueryLength = 2
    const val MedMetaMaxLength = 2000
    const val MedDoseAmountMax = 9999
    const val TodayBannerAutoAdvanceMs = 5000L
    const val FamilyFeedPreviewCount = 3
    const val FamilyFeedWindowDays = 7
}
